ParkingLotApi/Services/ParkingLotService.cs

```diff
@@ -40,5 +40,10 @@ public async Task<ParkingLot> GetByIdAsync(string id)
         {
             return await parkingLotsRepository.GetParkingLotById(id);
         }
+
+        public async Task<ParkingLot> PutAsync(string id, int capacity)
+        {
+            return await parkingLotsRepository.PutParkingLot(id, capacity);
```

Comment:
Should first get the object by id , then replace the capacity, then go to repository level update the object.